core/src/main/resources/hudson/PluginManager/_table.css

```diff
@@ -2,6 +2,7 @@
     background: white url('search.gif') no-repeat 2px center;
     padding-left: 20px;
     width: 15em;
+    font-size: 1em;
```

Comment:
I think this is a case for using the font-size in rems. ems should only be used when the element size depends on where they are (such as hyperlinks or inline monospaced elements)

Recommended font sizes are located on the `font.less` file